src/hotspot/cpu/aarch64/c1_LIRAssembler_aarch64.cpp

```diff
@@ -2559,7 +2559,7 @@ void LIR_Assembler::emit_lock(LIR_OpLock* op) {
   Register hdr = op->hdr_opr()->as_register();
   Register lock = op->lock_opr()->as_register();
   if (LockingMode == LM_MONITOR) {
-    if (op->info() != NULL) {
+    if (op->info() != null) {
```

Comment:
Hmmm... other places in the same file compare `op->info()` with `nullptr` and not `null`.

Reply:
It's also interesting that it compiled :-) What is 'null' anyway?
In any case, I am doing a scan of the whole patch and look for any possible re-introduction of NULL or even null.